script.service.hue/resources/lib/AmbiGroup.py

```diff
@@ -85,8 +87,13 @@ def resumeLightState(self):
             try:
                 self.bridge.lights[L].state(xy=xy, bri=bri, on=on, transitiontime=self.resume_transition)
             except QhueException as exc:
-                xbmc.log("[script.service.hue] onPlaybackStopped: Hue call fail: {}".format(exc))
-                reporting.process_exception(exc)
+                if exc.type_id == 201:  # 201 Param not modifiable because light is off error. 901: internal hue bridge error.
+                    pass
```

Comment:
**note:** I noticed this error during testing, when the lights are powered off I'd get an error dialog in Kodi when stopping playback. I remember this working fine, but I saw you upgraded Qhue and made some changes to the error handling so maybe this slipped in recently in `main`?

Let me know if this seems correct, also happy to extract this fix into a separate PR!

Reply:
Good catch, thanks!